packages/next/src/lib/metadata/generate/opengraph.tsx

```diff
@@ -202,7 +203,20 @@ export function OpenGraphMetadata({
       <Meta property="og:locale" content={openGraph.locale} />
       <Meta property="og:country_name" content={openGraph.countryName} />
       <Meta property="og:ttl" content={openGraph.ttl?.toString()} />
-      <MultiMeta propertyPrefix="og:image" contents={openGraph.images} />
+      {openGraph.images?.map((image, index) => (
+        <React.Fragment key={index}>
+          <Meta
+            property="og:image"
+            content={isStringOrURL(image) ? image : image.url}
+          />
+          {!isStringOrURL(image) && (
+            <MultiMeta
+              propertyPrefix="og:image"
+              contents={[{ ...image, url: null }]}
```

Comment:
This omits rendering of `og:image:url`.
Should we render both of `og:image` and `og:image:url`?